### DISCUSSION TITLE
Honeywell Smille SDC 8-21  - software V1.0  equivalent ?

### DISCUSSION THREAD

**Discussion by Bertsl** [phiresky/ebv-theta-to-mqtt#2] [Q&A]
Hello ,

I have another regulator made by Honeywell. The interface has almost the same functionality but I'm not 100% sure. Do you think the protocol is the same? I searched a lot on the web but not so easy to find.
I see in the photo the wiring to the Theta cache, could you give-me the number of the pins used for the serial converter?

I also tested a remote thermostat that looked very similar to the one Honewell sold (SDW20). It is a Theta RS-L but unfortunately not everything is perfect. Orders do not pass and some information is incorrect. Perhaps normal the software is newer in the Theta RS-L. Software V3.0...

Thanks for your help

**Comment by phiresky** [comment]
Hey! I don't think there's any way I could tell whether or not the software is the same or similar. You'll have to try it. From pictures it looks like it has the same debug port behind the panel on the front.

> could you give-me the number of the pins used for the serial converter

It's in the Readme:

> Connect RX to the third pin from the left and TX to the fifth pin from the left. (Disclaimer: I'm not sure which PIN is actually the TX pin, but this one works to ground (?) the RX connection at least).

Let me know if you try it or if I can help more ;)

**Comment by Bertsl** [comment]
> Hello ,
> 
> I have another regulator made by Honeywell. The interface has almost the same functionality but I'm not 100% sure. Do you think the protocol is the same? I searched a lot on the web but not so easy to find. I see in the photo the wiring to the Theta cache, could you give-me the number of the pins used for the serial converter?
> 
> I also tested a remote thermostat that looked very similar to the one Honewell sold (SDW20). It is a Theta RS-L but unfortunately not everything is perfect. Orders do not pass and some information is incorrect. Perhaps normal the software is newer in the Theta RS-L. Software V3.0...
> 
> Thanks for your help

[en2h0221-ge51r0913.pdf](https://github.com/phiresky/ebv-theta-to-mqtt/files/11200577/en2h0221-ge51r0913.pdf)
[0450017016_1004_30_FA_THETA_NRS_EbV_GB_Druck.pdf](https://github.com/phiresky/ebv-theta-to-mqtt/files/11200611/0450017016_1004_30_FA_THETA_NRS_EbV_GB_Druck.pdf)
